app/components/notification-list/template.hbs

```diff
@@ -0,0 +1,32 @@
+<div class="container">
+    <div class="row">
+        <div class="notification-list col-md-8">
+            <div class="notification-list-heading-container">
+                <h4 class="notification-list-heading">{{t 'components.notificationList.heading'}}</h4>
+                <div class="helper-text">
+                    {{t 'components.notificationList.helperText'}}
+                    <a href={{serviceLinks.settingsNotifications}}>
+                        {{t 'components.notificationList.userSettings'}}
+                    </a>.
+                </div>
+            </div>
+
+            <div class="line-separator"></div>
+            {{#if fetchData.isRunning}}
+                {{#content-placeholders as |placeholder|}}
+                    <div class="notification-title--skeleton">
```

Comment:
The placeholder isn't centered within the row (partly because there is a margin):
![screen shot 2018-04-04 at 10 01 16 am](https://user-images.githubusercontent.com/7131985/38312571-1557bb20-37f0-11e8-9f92-1f09889ffcef.png)

![screen shot 2018-04-04 at 10 00 49 am](https://user-images.githubusercontent.com/7131985/38312600-2bf79652-37f0-11e8-8b83-39706fa99c6c.png)